fastlane/Fastfile

```diff
@@ -45,3 +45,7 @@ platform :android do
     )    
   end
 end
+
+def truncate(string, max)
+  string.length > max ? "#{string[0...max]}" : string
+end
```

Comment:
if you know how to use this https://rdoc.info/gems/fastlane/String:truncate rather than making our own, please let me know!